src/runes.rs

```diff
@@ -736,6 +949,52 @@ mod tests {
     );
   }
 
+  #[test]
+  fn etched_reserved_rune_is_burned_if_an_unrecognized_even_tag_is_encountered() {
+    let context = Context::builder()
+      .arg("--index-runes-pre-alpha-i-agree-to-get-rekt")
+      .build();
+
+    context.mine_blocks(1);
+
+    let txid0 = context.rpc_server.broadcast_tx(TransactionTemplate {
+      inputs: &[(1, 0, 0, Witness::new())],
+      op_return: Some(
+        Runestone {
+          edicts: vec![Edict {
+            id: 0,
+            amount: u128::max_value(),
+            output: 0,
+          }],
+          etching: Some(Etching::default()),
+          burn: true,
```

Comment:
Instead of setting burn, maybe explicitly add an unrecognized even tag. I know this burn thing does that but maybe being explicit here helps test the space of unrecognized even tags.